x-pack/plugins/secops/public/containers/kpi_events/index.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getOr } from 'lodash/fp';
+import React from 'react';
+import { Query } from 'react-apollo';
+import { pure } from 'recompose';
+
+import { Direction, GetKpiEventsQuery, KpiItem } from '../../graphql/types';
+import { inputsModel } from '../../store';
+import { kpiEventsQuery } from './index.gql_query';
+
+export interface KpiEventsArgs {
+  id: string;
+  kpiEventType: KpiItem[];
+  loading: boolean;
+  refetch: inputsModel.Refetch;
+}
+
+export interface OwnProps {
+  children?: (args: KpiEventsArgs) => React.ReactNode;
+  id?: string;
+  filterQuery?: string;
+  poll: number;
+  sourceId: string;
+  startDate: number;
+  endDate: number;
+}
+
+export const KpiEventsQuery = pure<OwnProps>(
+  ({ children, filterQuery, id = 'kpiEventsQuery', poll, sourceId, startDate, endDate }) => (
+    <Query<GetKpiEventsQuery.Query, GetKpiEventsQuery.Variables>
+      query={kpiEventsQuery}
+      fetchPolicy="cache-and-network"
+      notifyOnNetworkStatusChange
+      pollInterval={poll}
+      variables={{
+        filterQuery,
+        sourceId,
+        pagination: {
+          limit: 0,
+          cursor: null,
+          tiebreaker: null,
+        },
+        timerange: {
+          interval: '12h',
+          from: startDate,
+          to: endDate,
+        },
+        sortField: {
+          sortFieldId: 'timestamp',
+          direction: 'descending' as Direction,
```

Comment:
I have seen this same `as Direction` cast used in a few other places and playing with it I think that they all should be typed as:

```ts
direction: Direction.descending
```

and that will remove another `as` cast and make it to where if it's changed to a different string you won't have to hunt and change all the occurrences.